drivers/spi/spi_xec_qmspi.c

```diff
@@ -0,0 +1,682 @@
+/*
+ * Copyright (c) 2019 Microchip Technology Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define LOG_LEVEL CONFIG_SPI_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(spi_xec);
```

Comment:
Please use LOG_MODULE_REGISTER(spi_xec, CONFIG_SPI_LOG_LEVEL) instead, and remove the LOG_LEVEL #define above.

Reply:
Done